### ISSUE TITLE
About the combination of batch size and number of gpus (a strange phenomenon)

### ISSUE THREAD

**Issue by lebron-2016** [DonDominic/MP2Net#10]
Dear author,

I recently tried to reproduce your results on NVIDIA 4090, and I found that when the batch size of a GPU is 2, it will exceed the memory. In order to keep consistent with your experimental settings, I used 4 4090 gpus (still keeping the batch size at 4, the initial learning rate at 1.25e-4, and all other settings are consistent with yours). However, the experimental results are very different. On the SatVideoDT dataset, MOTA is 16 points lower than your results. The specific results are shown below. Have you encountered such a situation before? How should this problem be solved?

![image](https://github.com/user-attachments/assets/4746ffa1-32df-41f0-b308-b1f119893de2)

Thanks for your help!!

**Comment by loazy** [comment]
My results were similar to yours, have you solved the problem since then, is this training without best weights I used the last training weights, this is a very urgent issue for me.
